.changeset/sixty-parents-repeat.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sl-design-system/spinner': major
+---
+
+- Removes the `variant` attribute; you can now only set the color of the spinner by changing the text color of the container (or the spinner itself) by css.
```

Comment:
Can you please update examples on the documentation website?
There are still some examples with `variant` left:

<img width="963" alt="image" src="https://github.com/user-attachments/assets/fc154ff6-8195-4f7f-a9eb-4f50e61fd70a" />

<img width="961" alt="image" src="https://github.com/user-attachments/assets/ce336852-2df8-423c-bff6-11a435de864c" />